JamJar/VideoPageViewController.swift

```diff
@@ -48,12 +54,21 @@ class VideoPageViewController: BaseViewController, UITableViewDelegate, UITableV
         artistsLabel.text = video.getArtistsString()
         venueLabel.text = concert.venue.name
         dateLabel.text = concert.date.string("MM-d-YYYY")
+        likesCountLabel.text = String((video.videoVotes.filter{$0.vote == 1}.first?.total)!)
```

Comment:
This is code-cloned from the jamjar VC I think.  Can you extract this "likesCount" out to the Video class? (and same with the dislikes)